source/components/ui/msm-modal/msm-modal.service.js

```diff
@@ -161,33 +161,23 @@
      *
      * @description
      *     Display a modal selection dialog.
-     * @param {string} title
-     *     The modal's title.
-     * @param {string} text
-     *     The modal's text.
      * @param {array} options
      *     The modal's selection options.
-     * @param {string=} size
-     *     The modal's size.
-     * @param {string=Ok} closeTitle
-     *     The label of the modal's confirm button.
-     * @param {string=Cancel} dismissTitle
-     *     The label of the modal's cancel button.
      */
-    function select(title, text, options, size, closeTitle, dismissTitle) {
-      return open({ options: options }, function ($modalInstance, options) {
+    function select(options) {
```

Comment:
What is the benefit of passing in an options object. It is now unclear which options exist for this API, esp. since the relevant documentation has been deleted... I think this is a degradation. Also. The API style now differs from the rest...